### PR TITLE
Merging bug fix change from PMS for PATRICStore

### PR DESCRIPTION
**Pull request by samseaver**
This PR is inconsequential, but necessary to maintain ObjectAPI between repositories

### CODE REVIEW

**Comment by samseaver** [comment]
Don't merge yet, running tests in docker

File: lib/fba_tools/fba_toolsImpl.pm [hunk 1]
```diff
@@ -4710,9 +4710,18 @@ sub bulk_export_objects
 		name => $output->{name},
 		description => $output->{description},
 	});
+    my $report_workspace = $params->{workspace};
+
+    #It's possible, using the API, to export objects from a workspace
+    #that the user doesn't own, and as such, cannot write the report to.
+
+    if(defined($params->{report_workspace})){
+	$report_workspace = $params->{report_workspace};
```
Comment (JamesJeffryes) [inline lib/fba_tools/fba_toolsImpl.pm:4721]: Should add this to the spec and recompile right?

File: lib/Bio/KBase/ObjectAPI/PATRICStore.pm [hunk 2]
```diff
@@ -688,7 +688,8 @@ sub save_model {
     }
     $summary->{status} = "complete";
     $summary->{status_timestamp} = Bio::KBase::utilities::timestamp();
-	$self->helper()->update_model_meta($ref,$summary);
+	#$self->helper()->update_model_meta($ref,$summary);
+	Bio::ModelSEED::patricenv::call_ws("update_metadata",{objects => [[$ref,$summary]]});
```
Comment (JamesJeffryes) [inline lib/Bio/KBase/ObjectAPI/PATRICStore.pm:692]: Why not fix the helper function?

File: lib/fba_tools/fba_toolsImpl.pm (hunk 1 above)
Comment (samseaver) [reply lib/fba_tools/fba_toolsImpl.pm:4721]: Good point

File: lib/Bio/KBase/ObjectAPI/PATRICStore.pm (hunk 2 above)
Comment (samseaver) [reply lib/Bio/KBase/ObjectAPI/PATRICStore.pm:692]: True.

Comment (samseaver) [reply lib/Bio/KBase/ObjectAPI/PATRICStore.pm:692]: Actually i changed my mind again, I remembered, patricenv is consistently called throughout PATRICStore.pm so I'd like to keep it this way.

**Review by JamesJeffryes — APPROVED** [review]
Looks OK to me

**Pull request merged**